src/bridges/layerZero/IexecLayerZeroBridge.sol

```diff
@@ -213,8 +222,17 @@ contract IexecLayerZeroBridge is
         // Calculate the amounts using the parent's logic (handles slippage protection)
         (amountSentLD, amountReceivedLD) = _debitView(amountLD, minAmountLD, dstEid);
 
-        // Burn the tokens from the sender's balance
-        BRIDGEABLE_TOKEN.crosschainBurn(from, amountSentLD);
+        if (block.chainid == 1) {
+            // Ethereum Mainnet: Transfer RLC tokens to LiquidityUnifier
+            // Workaround for Stargate UI compatibility - UI approves this contract
+            // instead of approving LiquidityUnifier directly
+            IRLCLiquidityUnifier(address(BRIDGEABLE_TOKEN)).RLC_TOKEN().safeTransferFrom(
+                from, address(BRIDGEABLE_TOKEN), amountSentLD
+            );
+        } else {
+            // Non-Ethereum chains: Burn tokens directly
+            BRIDGEABLE_TOKEN.crosschainBurn(from, amountSentLD);
```

Comment:
There are more calls to  `address(BRIDGEABLE_TOKEN)` then calls to `BRIDGEABLE_TOKEN.crosschain*()`. We can save `BRIDGEABLE_TOKEN` as an address and use `IERC7802(BRIDGEABLE_TOKEN)..` when needed.